test/functional/fixtures/api/es-next/test-controller/testcafe-fixtures/helpers.js

```diff
@@ -0,0 +1,3 @@
+export function missingAwaitFn (t) {
```

Comment:
We really need export this function to another file? Now it used only ones, will we use it again later?